pkg/operator/ceph/cluster/mon/mon.go

```diff
@@ -560,7 +562,19 @@ func (c *Cluster) startMon(m *monConfig, hostname string) error {
 		if !errors.IsAlreadyExists(err) {
 			return fmt.Errorf("failed to create mon %s. %+v", m.ResourceName, err)
 		}
-		logger.Debugf("mon %s already exists", m.ResourceName)
+		logger.Infof("deployment for mon %s already exists. updating if needed", m.ResourceName)
```

Comment:
could you change this to debug? it could be rather verbose with all the looping over the mons.